src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/IncrementalHash.net46.cs

```diff
@@ -116,7 +114,7 @@ public byte[] GetHashAndReset()
                 _hash.Initialize();
             }
 
-            _hash.TransformFinalBlock(s_empty, 0, 0);
+            _hash.TransformFinalBlock(Array.Empty<byte>(), 0, 0);
```

Comment:
Hmm.  We build a .net46.cs file against .NET 4.5 surface area in TFS?  That seems... wrong. @weshaggard, are you aware of any remaining issues here?  If we're concerned about it, I can revert the change to this file / add a comment citing why it was done.  But it'd be better if we didn't need to worry about such internal compilation issues when updating public sources.